zeebe/exporters/camunda-exporter/src/main/java/io/camunda/exporter/ExporterComponentsProvider.java

```diff
@@ -0,0 +1,18 @@
+/*
+ * Copyright Camunda Services GmbH and/or licensed to Camunda Services GmbH under
+ * one or more contributor license agreements. See the NOTICE file distributed
+ * with this work for additional information regarding copyright ownership.
+ * Licensed under the Camunda License 1.0. You may not use this file
+ * except in compliance with the Camunda License 1.0.
+ */
+package io.camunda.exporter;
+
+import io.camunda.exporter.schema.descriptors.IndexDescriptor;
+import io.camunda.exporter.schema.descriptors.IndexTemplateDescriptor;
+import java.util.List;
+
+public interface ExporterComponentsProvider {
```

Comment:
:x: Add java docs for the interface and the methods.

Reply:
:wrench: Also if you can think of a better name that would be great :smile: I suggested this without thinking much.